main.go

```diff
@@ -1,6 +1,7 @@
 package main
 
 import (
+	"github.com/PDOK/gokoala/ogc/processes"
 	"log"
```

Comment:
minor linting dingetjes, gebruiken de goimports volgorde kan je ook in intellij instellen: zie https://github.com/PDOK/gokoala#intellij--goland

Reply:
fixed